.eslintrc.js

```diff
@@ -85,6 +85,13 @@ module.exports = {
       },
     ],
     'prefer-promise-reject-errors': 0,
+    'shopify/jsx-no-hardcoded-content': [
+      'warn',
```

Comment:
Ideally, we should make this an `error` rather than a warning as if we are going to user rules I'd rather they be enforced as a standard rather than left as something that cal slip through the cracks unaddressed.

Reply:
@mrfelton I set it as a error at first but because I still had unresolved issues, I couldn't push my branch out lol. I will keep it as a warning until all issues are resolved.